### ISSUE TITLE
how to cite?

### ISSUE THREAD

**Issue by ekg** [marschall-lab/GFAffix#9]
What's your preferred method for citation?

**Comment by danydoerr** [comment]
I'm uploading a preprint. Will let you know the link very soon.

**Comment by ekg** [comment]
We should add it to the HPRC paper.

Doing the same for PGGB, which led to this request.

**Comment by glennhickey** [comment]
Yeah, it should definitely go in the HPRC paper -- I'll also update update the minigraph-cactus paper which is just citing the website here at the moment.

**Comment by danydoerr** [comment]
> I'm uploading a preprint. Will let you know the link very soon.

That was a bit too optimistic. For now, please cite the HPRC paper, a preprint version will follow some time soon.
